index.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  configs: {
+    tslint: require('./lib/config/all.js'),
```

Comment:
Also, meta note: is the name of this repo right? Are custom `tslint` configs usually named `tslint-config-foo` or `tslint-plugin-foo`? We went with plugin for `eslint` because we had to, might not have to here though.